config.yml

```diff
@@ -37,3 +37,7 @@ genesis:
     evm:
       params:
         evm_denom: "aevmos"
+    ica:
+      params:
+        allow_messages: ["/cosmos.staking.v1beta1.MsgDelegate", "/cosmos.gov.v1beta1.MsgVote"]
```

Comment:
this is for Ignite CLI and should be deprecated. Can you delete this file altogether?

Reply:
consider gov v1 msgs, e.g `/cosmos.gov.v1.MsgVote`